lib/pdk/util.rb

```diff
@@ -58,8 +58,13 @@ def package_install?
     end
     module_function :package_install?
 
+    def development_mode?
+      PDK::VERSION.end_with? 'pre'
+    end
+    module_function :development_mode?
```

Comment:
Should this maybe test if the code is part of a git checkout or something instead?

Reply:
It could, but that could complicate matters if we (or others) attempt to run a tag from source for a repro or something? I think ultimately, the simplicity of just checking whether we're on an "official" version or a dev/preview version is more explicit. I could go either way.